internal/gapicgen/cmd/genbot/github.go

```diff
@@ -178,13 +179,15 @@ func (gc *GithubClient) GetRegenPR(ctx context.Context, repo string, status stri
 // CreateGenprotoPR creates a PR for a given genproto change.
 //
 // hasCorrespondingPR indicates that there is a corresponding google-cloud-go PR.
-func (gc *GithubClient) CreateGenprotoPR(ctx context.Context, genprotoDir string, hasCorrespondingPR bool) (prNumber int, _ error) {
+func (gc *GithubClient) CreateGenprotoPR(ctx context.Context, genprotoDir string, hasCorrespondingPR bool, changes []*generator.ChangeInfo) (prNumber int, _ error) {
 	log.Println("creating genproto PR")
-
-	body := genprotoCommitBody
+	var bodyBuilder strings.Builder
+	bodyBuilder.WriteString(genprotoCommitBody)
 	if !hasCorrespondingPR {
-		body += "\n\nThere is no corresponding google-cloud-go PR.\n"
+		bodyBuilder.WriteString("\n\nThere is no corresponding google-cloud-go PR.\n")
+		bodyBuilder.WriteString(formatChanges(changes, false))
 	}
+	sBody := bodyBuilder.String()
```

Comment:
Minor: meant rename the builder so you can name this `body`. `sb` is used elsewhere for the builder if you want to be consistent+short.

Reply:
done.